src/orga/subteams.rst

```diff
@@ -122,3 +122,58 @@ Members
 - Anthony Scopatz <scopatz@gmail.com>
 - Christian Roth <ch.m.roth@gmail.com>
 - Lori A. Burns <lori.burns@gmail.com>
+
+
+Staging Sub-Team
+================
+
+Role
+----
+
+Review and merge feedstock candidates in the staged-recipes repository. Help users to create conda-forge compatible recipes.
+
+
+Charter
+-------
+Dynamic
+
+Responsibility
+--------------
+
+Introducing a recipe for most users is the first step of becoming involved with the development of conda-forge.
+Especially for new maintainers it is crucial to be able to ask questions and receive helpful and constructive feedback.
+
+The staging team is responsible for:
+
+ - reviewing and merging pull requests in conda-forge/staged-recipes
+ - answering questions and giving feedback regarding conda-forge requirements
+ - identifying common misconceptions and problems due to unclear documentation
+ - help the documentation team maintain clear documentation that simplifies contributing packages
+
```

Comment:
One thing I have noticed is that many feedstock maintainers are starting to ping members of the staged-recipes language specific subteams (e.g., conda-forge/help-python) for help with maintenance on existing feedstocks. This actually works pretty well and takes some of the load off of core. Since this PR will need a formal vote by core, I would say we should formalize this responsibility at the same time.

Here is draft language (at the top level and the list):

```
Further, maintainers of existing feedstocks sometimes run into technical issues with recipe maintenance and bug fixes. 

 - assist existing feedstock maintainers with questions/issues that arise during regular recipe maintenance and bug fixes
```

Reply:
What about: 
- assist core in supporting feedstock maintainers when questions/issues arise during recipe maintenance.
I think we could add that to the list as it is.

Reply:
LGTM!